styles/main.css

```diff
@@ -0,0 +1,651 @@
+* {
+  box-sizing: border-box;
+}
+
+/* FONTS */
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-regular-web.woff") format("woff");
+  font-weight: normal;
+}
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-medium-web.woff") format("woff");
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-bold-web.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-heavy-web.woff") format("woff");
+  font-weight: 800;
+}
+
+body,
+html {
+  margin: 0;
+  font-family: "chirp";
+  font-weight: normal;
+  overscroll-behavior-y: none;
+  --gray: #f6f9f9;
+  --blue: #1da1f2;
+  --border-color: rgba(128, 128, 128, 0.145);
+}
+
+/* --------------------------------*/
+.font-chirp {
+  font-family: "chirp";
+}
+.h-100 {
+  height: 100%;
+}
+
+.w-100 {
+  width: 100%;
+}
+.twitter-color {
+  color: var(--blue);
+  fill: var(--blue);
+}
+
+.gray {
+  color: gray;
+}
+.il-b {
+  display: inline-block;
+}
+.b-0 {
+  border: none;
+}
+.b-light {
+  border: 2px solid var(--border-color);
+}
+
+.b-thin {
+  border: 1px solid var(--border-color);
+}
+.b-b-thin {
+  border-bottom: 1px solid var(--border-color);
+}
+.b-sd-thin {
+  border-left: 1px solid var(--border-color);
+  border-right: 1px solid var(--border-color);
+}
+
+.b-eg-thin {
+  border-top: 1px solid var(--border-color);
+  border-bottom: 1px solid var(--border-color);
+}
+
+.b-white {
+  border-color: white;
+}
+
+.b-r-sm {
```

Comment:
some class names don't really tell me what they represent, so it is slightly hard to immediately know what they do (esp. when applied in HTML) unless I come back here to the file and look at the definition.

For readability could be good to start with long-form class names `.border-rad-sm` although this does increase the amount of data transferred on page load.

An approach could be to break the CSS into some sections?

Reply:
Noted ! Maybe I could create multiple files and keep the 'general' css on one so i can actually keep adding to it during the course